coffea/jetmet_tools/JetCorrectionUncertainty.py

```diff
@@ -1,8 +1,7 @@
 from ..lookup_tools.jec_uncertainty_lookup import jec_uncertainty_lookup
 import warnings
 import re
-from ..util import awkward
-from ..util import awkward1
+import awkward as ak
 from ..util import numpy as np
```

Comment:
This can be just `import numpy as np` (or maybe `import numpy` per previous comments) right? I believe that if we do in the future try for `cupy` et al we'll have an alternative implementation (awkward1 has approached this differently, deciding the library in the dunder method callback)